specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.1-preview.5/TextAnalytics.json

```diff
@@ -2664,6 +2667,15 @@
     }
   },
   "parameters": {
+    "ApiVersion": {	
+      "name": "ApiVersion",
+      "description": "Text Analytics API version (for example, v3.0).",
+      "x-ms-parameter-location": "client",
+      "required": true,
+      "type": "string",
```

Comment:
This should be a string-typed enum with a single value (the API version in question). As it is for query parameter based versions.

Reply:
Is something like 

```
"type": "string",
"enum": [
       // all the version supported?
],
```

Reply:
@johanste - I'm not sure if it's necessary - I believe AutoRest identities the global parameter called "ApiVersion" and pre-populate it with a constant value defined in the root level `"version"`.

Reply:
Thanks @jianghaolu - that is good to know. I did not realize that autorest (presumably modeler 4?) did this automatically. 

I would still suggest adding the enum/single string value since there is, in fact, only one string value that this specific specification supports. If nothing else, it would help tools other than autorest (e.g. docfx) determine what the correct value is.

Reply:
Thank you. @johanste. Does it look right to you?